[BUG] does not work with electron sample

**Context:**
- Playwright Version: `1.14.1`
- Operating System: macOS ARM
- Node.js version: `v16`
- Browser: Electron
- Extra: [any specific details about your environment]

**Code Snippet**

Help us help you! Put down a short code snippet that illustrates your bug and
that we can run and debug locally. For example:

```javascript

(async () => {
    const { _electron } = require('playwright');
    
    // Launch Electron app.
    const electronApp = await _electron.launch({ args: ['main.js'] });

    // Evaluation expression in the Electron context.
    const appPath = await electronApp.evaluate(async ({ app }) => {
        // This runs in the main Electron process, parameter here is always
        // the result of the require('electron') in the main app script.
        return app.getAppPath();
    });
    console.log(appPath);

    // Get the first window that the app opens, wait if necessary.
    const window = await electronApp.firstWindow();
    // Print the title.
    console.log(await window.title());
    // Capture a screenshot.
    await window.screenshot({ path: 'intro.png' });
    // Direct Electron console to Node terminal.
    window.on('console', console.log);
    // Click button.
    await window.click('text=Click me');
    // Exit app.
    await electronApp.close();
})();
```

**Describe the bug**

Output:

```
~/Desktop/playwright> node index.js
node:internal/process/promises:246
          triggerUncaughtException(err, true /* fromPromise */);
          ^

electron.launch: Cannot find module 'electron/index.js'
Require stack:
- /Users/bpasero/Desktop/playwright/node_modules/playwright/lib/server/electron/electron.js
- /Users/bpasero/Desktop/playwright/node_modules/playwright/lib/server/playwright.js
- /Users/bpasero/Desktop/playwright/node_modules/playwright/lib/inprocess.js
- /Users/bpasero/Desktop/playwright/node_modules/playwright/index.js
- /Users/bpasero/Desktop/playwright/index.js
    at /Users/bpasero/Desktop/playwright/index.js:5:41
    at Object.<anonymous> (/Users/bpasero/Desktop/playwright/index.js:27:3)
    at Module._compile (node:internal/modules/cjs/loader:1108:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
    at Module.load (node:internal/modules/cjs/loader:988:32)
    at Function.Module._load (node:internal/modules/cjs/loader:828:14)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)
    at node:internal/main/run_main_module:17:47 {
  name: 'Error'
}
```

--- a/packages/playwright-core/src/server/electron/electron.ts
+++ b/packages/playwright-core/src/server/electron/electron.ts
@@ -24,6 +24,7 @@ import { CRExecutionContext } from '../chromium/crExecutionContext';
 import * as js from '../javascript';
 import { Page } from '../page';
 import { TimeoutSettings } from '../../utils/timeoutSettings';
+import { wrapInASCIIBox } from '../../utils/utils';
 import { WebSocketTransport } from '../transport';
 import { launchProcess, envArrayToObject } from '../../utils/processLauncher';
 import { BrowserContext, validateBrowserContextOptions } from '../browserContext';
@@ -134,12 +135,32 @@ export class Electron extends SdkObject {
 
       const browserLogsCollector = new RecentLogsCollector();
       const env = options.env ? envArrayToObject(options.env) : process.env;
+
+      let command: string;
+      if (options.executablePath) {
+        command = options.executablePath;
+      } else {
+        try {
+          // By default we fallback to the Electron App executable path.
+          // 'electron/index.js' resolves to the actual Electron App.
+          command = require('electron/index.js');
+        } catch (error: any) {
+          if ((error as NodeJS.ErrnoException)?.code === 'MODULE_NOT_FOUND') {
+            throw new Error('\n' + wrapInASCIIBox([
+              'Electron executablePath not found!',
+              'Please install it using `npm install -D electron` or set the executablePath to your Electron executable.',
+            ].join('\n'), 1));
+          }
+          throw error;
+        }
+      }
+
       // When debugging Playwright test that runs Electron, NODE_OPTIONS
       // will make the debugger attach to Electron's Node. But Playwright
       // also needs to attach to drive the automation. Disable external debugging.
       delete env.NODE_OPTIONS;
       const { launchedProcess, gracefullyClose, kill } = await launchProcess({
-        command: options.executablePath || require('electron/index.js'),
+        command,
         args: electronArguments,
         env,
         log: (message: string) => {

